server/svix-server/src/db/models/application.rs

```diff
@@ -60,13 +149,56 @@ impl ActiveModelBehavior for ActiveModel {
     }
 }
 
+impl ActiveModel {
+    pub fn new_for_org(org_id: OrganizationId) -> Self {
+        Self {
+            org_id: Set(org_id),
+            ..Self::new()
+        }
+    }
+
+    pub async fn get_with_application_metadata(
+        db: &DatabaseConnection,
+        org_id: OrganizationId,
+        id_or_uid: ApplicationIdOrUid,
+    ) -> error::Result<Option<(Self, applicationmetadata::ActiveModel)>> {
+        let models =
+            ctx!(Model::get_with_application_metadata(db, org_id.clone(), id_or_uid).await)?;
+
+        let result = models.map(|(app, metadata)| {
+            let metadata = metadata.map(|m| m.into()).unwrap_or_else(|| {
+                applicationmetadata::ActiveModel::new_for_app(app.id.clone(), None)
+            });
+            (app.into(), metadata)
+        });
+
+        Ok(result)
+    }
+}
+
 impl Entity {
     pub fn secure_find(org_id: OrganizationId) -> Select<Entity> {
         Self::find()
             .filter(Column::OrgId.eq(org_id))
             .filter(Column::Deleted.eq(false))
     }
 
+    pub fn secure_find_many(
```

Comment:
Secure find already does many, so the name is confusing, though I don't fully understand the reason for it to exist. It's logic that should only exist in one place (list apps), so why abstract it?